code/controllers/master.dm

```diff
@@ -148,7 +148,7 @@ var/global/datum/controller/master/Master = new()
 				//this is a queue for any SS skipped or paused for tick reasons, to be ran first next tick
 				if (priority_queue.len)
 					priorityrunning = priority_queue.len
-					subsystemstorun = priority_queue | subsystems
+					subsystemstorun = priority_queue | subsystemstorun
```

Comment:
this is how it was suppose to be, so that if the MC paused while running subsystems, it resumed at the same spot (after running priority subsystems) but i messed up and had it copy from the wrong list.